courses/free-courses-en.md

```diff
@@ -1,13 +1,14 @@
 ### Index
 
-* [Index](#index)
+
```

Comment:
Maybe I should have said "remove line" in https://github.com/EbookFoundation/free-programming-books/pull/5578#discussion_r719655588 😶‍🌫️😣

If applied, it prevents conflicts with other PR #5581